api/src/Controller/BaseController.php

```diff
@@ -138,6 +138,11 @@ abstract class BaseController
     */
     protected $currentEvent = -1;
 
+    /**
+     * @var bool
+    */
+    public $internal = false;
```

Comment:
What are some other things that `$internal` would apply to? With members I can see it was in the FindMembers controller, are there any other immediate examples?

Reply:
Not that I know about here.  But moving it into here makes it easier to set in `IncludeResource` and check in `GetMember` Otherwise we have to see if the property exists before setting it or checking it and it just felt cleaner to move it into the Base.